lib/core/GraphicsFactory.js

```diff
@@ -92,10 +92,12 @@ GraphicsFactory.prototype._clear = function(gfx) {
  *   <g class="djs-children"></g>
  * </g>
  *
- * @param {Object} parent
  * @param {String} type the type of the element, i.e. shape | connection
+ * @param {Snap<SVGElement>} [childrenGfx]
```

Comment:
We're not returning Snap<SVGElement> here, instead only raw SVG elements.

Reply:
Good old times.

Reply:
:slightly_smiling_face: :old_key:

Reply:
updated the return docs.